src/odemis/gui/model/__init__.py

```diff
@@ -119,6 +119,7 @@ class MainGUIData(object):
     # Mapping between the component role and the attribute name on the MainGUIData
     _ROLE_TO_ATTR = {
         "ccd": "ccd",
+        # ccd* -> ccds[]
```

Comment:
what is this line for?

Reply:
Just to remember that all the "ccd*" roles get mapped to ccds. They is a special code that does it, so it cannot be mapped as the standard roles.